package.json

```diff
@@ -11,7 +11,7 @@
   "author": "Xander Castillo",
   "dependencies": {
     "axios": "^0.26.1",
-    "discord.js": "^14.7.1",
+    "discord.js": "^14.8.0",
```

Comment:
Major: May I know why we need to bump the version up to v14.8.0? Before restarting the bot, we need to properly install the packages when we update versions or add new dependencies.

Reply:
Thanks for this. We'll take note of this when we release the next version.